org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/LanguageServersTest.java

```diff
@@ -778,6 +789,81 @@ public void testGetDocument() throws Exception {
 
 		assertEquals(document, executor.getDocument());
 	}
+	
+	@Test
+	public void testVersionSupportSuccess() throws Exception {
```

Comment:
Since we have moved `VersionedEdits` out of `LanguageServers`, I would create as well a new test class for this two methods

Reply:
OK